### PR TITLE
docs: improve `Path` documentation

### PR DESCRIPTION
**Pull request by squigglesdev**
Improved the clarity of the docs for the `Path` component.

### CODE REVIEW

File: packages/docs/docs/components/path.mdx [hunk 1]
```diff
@@ -30,17 +30,43 @@ export default makeScene2D(function* (view) {
 });
 ```
 
-This component allows us to draw and animate SVG path.
+The `Path` allows us to draw and animate SVG paths.
```
Comment (aarthificial) [inline packages/docs/docs/components/path.mdx:33]:
```suggestion
The `Path` node allows us to draw and animate SVG paths.
```

File: packages/docs/docs/components/path.mdx [hunk 2]
```diff
@@ -30,17 +30,43 @@ export default makeScene2D(function* (view) {
...
 
-## Defining Path
+:::tip
 
-You can define path to display by setting `data` property of Path component.
+Path nodes are great for displaying pre-calculated SVG paths, but the format can be confusing. If you want to have more
+intuitive control of the path, and control each point within Motion Canvas, we reccomend checking out the [`Spline`][spline] component.
```
Comment (aarthificial) [inline packages/docs/docs/components/path.mdx:38]:
```suggestion
`Path` nodes are great for displaying precalculated SVG paths, but the format can be confusing. If you want to have more
intuitive control of the path, and control each point within Motion Canvas, we recommend checking out the [`Spline`][spline] component.
```

File: packages/docs/docs/components/path.mdx [hunk 3]
```diff
@@ -30,17 +30,43 @@ export default makeScene2D(function* (view) {
...
 
-## Animating Path
+:::
 
-### Morphing Between Two Path
+## Defining paths
 
-To morph between two path you can tween `data` property.
+We can specify the path to be displayed by setting the data property of the Path component:
```
Comment (aarthificial) [inline packages/docs/docs/components/path.mdx:44]:
```suggestion
We can specify the path to be displayed by setting the `data` property of the `Path` node:
```

File: packages/docs/docs/components/path.mdx [hunk 4]
```diff
@@ -30,17 +30,43 @@ export default makeScene2D(function* (view) {
...
+
+  view.add(
+    <Path
+      ref={path}
+      lineWidth={4}
+      stroke={'#e13238'}
+      data="M301.113,12.011L576.715,584.766L25.508,584.766L301.113,12.011z"
+      position={[-100, -100]}
+      scale={0.25}
+    ></Path>,
```
Comment (aarthificial) [inline packages/docs/docs/components/path.mdx:61]:
```suggestion
    />,
```

File: packages/docs/docs/components/path.mdx [hunk 5]
```diff
@@ -67,10 +93,10 @@ export default makeScene2D(function* (view) {
 });
 ```
 
-### Animating objects along a spline
+### Animating objects along a path
 
-Like Spline and Bezier, you can also make other components follow this component
-path. You can use the `getPointAtPercentage` to get point along this path.
+Similar to the [`Spline`][spline] and [`Bézier curve`][bezier] components, the `Path` component has
+a `getPointAtPercentage` method that can be used to animate an object along it.
```
Comment (aarthificial) [inline packages/docs/docs/components/path.mdx:99]: I'd change this to something like:
```suggestion
Like all nodes extending the [`Curve`][curve] class, the `Path` node has
a `getPointAtPercentage` method that can be used to animate an object along it:
```

**Review by aarthificial — CHANGES_REQUESTED** [review]

**Comment by squigglesdev** [comment]
I've added your suggestions, and actually remembered to run prettier this time 😅

File: packages/docs/docs/components/path.mdx [hunk 6]
```diff
@@ -30,17 +30,46 @@ export default makeScene2D(function* (view) {
...
+  view.add(
+    <Path
+      ref={path}
+      lineWidth={4}
+      stroke={'#e13238'}
+      data="M301.113,12.011L576.715,584.766L25.508,584.766L301.113,12.011z"
+      position={[-100, -100]}
+      scale={0.25}
+    />,
+  );
```
Comment (aarthificial) [inline packages/docs/docs/components/path.mdx:65]: This codeblock contains invalid TS code which is why the verification is not passing.
You're missing the closing brackets:
```suggestion
  );
});
```

File: packages/docs/docs/components/path.mdx [hunk 7]
```diff
@@ -67,10 +96,10 @@ export default makeScene2D(function* (view) {
 });
 ```
 
-### Animating objects along a spline
+### Animating objects along a path
 
-Like Spline and Bezier, you can also make other components follow this component
-path. You can use the `getPointAtPercentage` to get point along this path.
+Like all nodes extending the [`Curve`][curve] class, the `Path` node has a
```
Comment (aarthificial) [inline packages/docs/docs/components/path.mdx:101]: ``[`Curve`][curve]`` is a [reference-style link](https://www.markdownguide.org/basic-syntax/#reference-style-links), it assumes that the actual link to `curve` is defined somewhere at the bottom of the file:
```md
[curve]: /api/2d/components/Curve
```

**Review by aarthificial — CHANGES_REQUESTED** [review]

**Review by aarthificial — APPROVED** [review]

**Comment by squigglesdev** [comment]
Whoops kinda forgot about this, thanks for finishing it off for me 😅

**Comment by aarthificial** [comment]
@squigglesdev No worries!

**Pull request merged**